testing/test_pytest_html.py

```diff
@@ -238,9 +238,11 @@ def pytest_runtest_makereport(item, call):
 
     @pytest.mark.parametrize('content, encoded', [
         ("u'\u0081'", 'woE='),
-        ("'foo'", 'Zm9v')])
+        ("'foo'", 'Zm9v'),
+        (b"\xe2\x80\x93" if PY3 else "'\xe2\x80\x93'", "4oCT")])
```

Comment:
There is only one reason for using a condition within a parameter in this case: different behavior of bytes string formatting in different Python versions. And In my honest opinion additional parameters for specific versions of Python in this case are redundantly.